origocli/commands/events.py

```diff
@@ -12,16 +13,29 @@ class EventsCommand(BaseCommand):
     __doc__ = f"""Oslo :: Events
 
 Usage:
-  origo events put <datasetid> <versionid> [--file=<file> options]
-  origo events stat <datasetid> [options]
+  origo events cat <dataset-uri> [options]
+  origo events create-stream <dataset-uri> [--skip-raw] [options]
+  origo events delete-stream <dataset-uri> [options]
+  origo events enable-subscription <dataset-uri> [options]
+  origo events disable-subscription <dataset-uri> [options]
+  origo events enable-sink <dataset-uri> --sink-type=<sink_type> [options]
+  origo events disable-sink <dataset-uri> --sink-id=<sink_id> [options]
+  origo events put <dataset-uri> [--file=<file> options]
+  origo events stat <dataset-uri> [options]
 
 Examples:
-  echo '{{"hello": "world"}}' | origo events put test-event 1
-  echo '[{{"hello": "world"}}, {{"world": "hello"}}]' | origo events put test-event 1
-  cat /tmp/event.json | origo events put test-event 1
-  origo events put test-event 1 --file=/tmp/event.json
-  origo events stat test-event
-  origo events stat test-event --format=json | jq ".last_hour.events"
+  origo events cat ds:my-dataset-id/1
+  origo events cat my-dataset-id/1
+  origo events cat my-dataset-id
+  origo events create-stream ds:my-dataset-id/1
+  origo events enable-sink ds:my-dataset-id/1 --sink-type=s3
```

Comment:
Er det ikke mer vanlig å angi parametere uten `=`, dvs. `--sink-type s3` ? Tror den forøvrig støtter begge varianter, men jeg er mest vant med den uten `=`.

Reply:
Ja, tror det (og docopt støtter varianter begge så vidt jeg kan se). Dette bør kanskje endres for samtlige "options" i en annen runde?

Reply:
Hm, jeg kan bare huske å ha sett `--longopt=value` varianten. Det er også [GNU-standarden](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html).

Reply:
> A long option may take a parameter, of the form --arg=param **or** --arg param
https://man7.org/linux/man-pages/man3/getopt.3.html

Har som nevnt fulgt opp eksisterende eksempler for bruk av `--longopt` - så mitt spørsmål er om man heller skal bestemme og gjøre om dette globalt som del av videre omstrukturering (uansett om man aksepterer begge eller ikke)? 

F.eks. sett litt på [typer](https://typer.tiangolo.com/), som også ser ut til å akseptere begge varianter, dog med `--longopt ARG` som default:
```
Usage: main.py datasets ls [OPTIONS]

  List datasets

Options:
  -f, --filter TEXT    Filter by dataset id
  -fmt, --format TEXT  Format: json, simple, grid, github  [default:
                       fancy_grid]

  --help               Show this message and exit
```

Reply:
Det er bibliotekfunksjonen `getopt`, ikke [brukerkommandoen](https://man7.org/linux/man-pages/man1/getopt.1.html):

> A long option normally begins with '--' followed by the long option
> name.  If the option has a required argument, it may be written
> directly after the long option name, separated by '=', or as the next
> argument (i.e., separated by whitespace on the command line).  If the
> option has an optional argument, it must be written directly after
> the long option name, separated by '=', if present [...]

Det gir mening, for når parameteren er valgfri blir det tvetydig uten `=`, men når det er påkrevd går det bra å velge å droppe `=`.

Reply:
Er bare jeg som er vant til å skrive uten `=` da :-D

Var egentlig bare en kommentar til eksempel/dokumentasjonen, men er helt greit sånn det står nå :)

Reply:
Åja, jeg trodde det var snakk om hva vi ville at kommandolinja skulle støtte. :)

For dokumentasjonen er vel begge deler greit, så lenge vi er konsistente. Men heller nok litt mot å beholde `=`, for da slipper vi å tenke over de tilfellene der parameteren er valgfri og `=` blir nødvendig.